index.html

```diff
@@ -11,21 +11,23 @@
       type="module"
     ></script>
   </head>
-  <body class="flex justify-center">
+  <body class="flex justify-center bg-gray-50">
     <div class="flex-grow max-w-2xl mt-10 px-4">
-      <header>
-        <h1 class="font-bold mb-10 text(center gray-600 xl)">
-          Koodikrapula.fi
-        </h1>
+      <header class="flex justify-center">
+        <a href="#">
+          <h1 class="font-bold mb-6 text(center gray-600 2xl) hover:bg-gray-100 hover:text-red-400 w-min p-3 rounded">
+            Koodikrapula.fi
+          </h1>
+        </a>
       </header>
 
       <main>
         <h2 class="sr-only">Viimeisin jakso</h2>
 
-        <article class="bg-gray-50 mb-12 -mx-4 p-6 md:rounded">
+        <article class="mb-12 -mx-4 p-6 rounded bg-white shadow-md">
```

Comment:
Nit:

IMO the rounded borders look silly in xs:

![image](https://user-images.githubusercontent.com/2226144/112726669-55264500-8f27-11eb-803e-48db061fe090.png)

Thus I would vote to change `rounded` to `md:rounded`. But I'll let you decide. 😄